scripts/setup-local.sh

```
@@ -0,0 +1,152 @@
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+ENV_FILE="$PROJECT_ROOT/.env"
+SEED_DIR="$PROJECT_ROOT/seed"
+DUMP_FILE="$SEED_DIR/dump.sql"
+SKIP_DUMP=false
+SKIP_SEED=false
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  SED_INPLACE=(-i '')
+else
+  SED_INPLACE=(-i)
```

Shellcheck (0.11.0)

[warning] SED_INPLACE appears unused. Verify use (or export if used externally).

(SC2034)